tests/ui/invalid_property_args.stderr

```diff
@@ -52,14 +52,14 @@ error[E0277]: `PhantomData<i32>` cannot be converted to a Python object
 45 |     value: ::std::marker::PhantomData<i32>,
    |            ^ required by `#[pyo3(get)]` to create a readable property from a field of type `PhantomData<i32>`
    |
-   = help: the trait `IntoPy<Py<PyAny>>` is not implemented for `PhantomData<i32>`, which is required by `PhantomData<i32>: PyO3GetField`
-   = note: implement `ToPyObject` or `IntoPy<PyObject> + Clone` for `PhantomData<i32>` to define the conversion
-   = note: required for `PhantomData<i32>` to implement `PyO3GetField`
-note: required by a bound in `PyClassGetterGenerator::<ClassT, FieldT, Offset, false, false>::generate`
+   = help: the trait `for<'py> IntoPyObject<'py>` is not implemented for `PhantomData<i32>`, which is required by `PhantomData<i32>: PyO3GetFieldIntoPyObject`
```

Comment:
A thought: from testing locally, if I change the definition of `PyO3GetFieldIntoPyObject` to have a lifetime `'py`, i.e. `PyO3GetFieldIntoPyObject<'py>: IntoPyObject<'py> + Clone`.

... then (after adjusting other uses accordingly), the error message looks more like this:

```suggestion
   = help: the trait `IntoPyObject<'_>` is not implemented for `PhantomData<i32>`, which is required by `for<'py> PhantomData<i32>: PyO3GetFieldIntoPyObject<'py>`
```

Is that better? I think it might be slightly because it moves the HRTB later in the message. I _think_ it's functionally equivalent otherwise?

Reply:
I think it should be equivalent to put the HRTB in the method bound instead of the the super trait, yes.